runelite-client/src/main/java/net/runelite/client/plugins/grounditems/config/ValueCalculationMode.java

```diff
@@ -0,0 +1,21 @@
+package net.runelite.client.plugins.grounditems.config;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum ValueCalculationMode
+{
+    HA("High Alchemy"), // calc highlight by HA value
+    GE("Grand Exchange"), // calc by GE
+    BOTH("Both"); // use whatever is highest.
```

Comment:
maybe this should be `HIGHEST`?

Reply:
I wasn't sure whether it should be called 'HIGHEST' or 'BOTH', but now that you mention it, I think it will be a bit clearer to the user if it's named 'HIGHEST'.